src/zeroconf/_services/browser.py

```diff
@@ -92,23 +98,68 @@
 
 _QuestionWithKnownAnswers = Dict[DNSQuestion, Set[DNSPointer]]
 
+heappop = heapq.heappop
+heappush = heapq.heappush
+
+
+class _ScheduledPTRQuery:
```

Comment:
we need units for this class since it implement dunder methods that _might_ be needed by `heapq` in the future but currently aren't used.  We have to implement them even if `__lt__` is the only one actually called now since `heapq` could change the implementation in the future and than this would break